CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Updated wokeignore to avoid unfixable warning
 - Update Chart.js to v4.4.3
 - Use tornado library version >= 6.4.1
+- Load only mei variants hitting demo panel genes in scout demo instance
```

Comment:
This isn't quite the thing that happened, is it? 😊 Either document the commands used in the VCF, or maybe say "Fewer variants in the MEI demo file"?

Reply:
"Fewer variants in the MEI demo file"  it is! 😆